gst/nnstreamer/tensor_common.h

```diff
@@ -51,14 +51,6 @@
 #define nns_memset memset
 #endif
 
-#ifdef NO_AUDIO
```

Comment:
If you want to use ifndef every time it might be required in code, you should delete no_audio_define.h file. However, we need discuss about which one we are going to use. Macro or ifdef. ( In my opinion, we need to reduce ifdefs in code cause we may need ifdefs for video and text or octet, too.  )

Reply:
no_audio_define.h removed.

I focused to move the functions and caps definition about media type into tensor-converter.
In tensor-converter, it just uses c-preprocessor (NO_AUDIO) to disable audio support.
We need to determine how to disable each media type later. (use preprocessor or set macros)